.github/workflows/test.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           PR_NUMBER: ${{ github.event.pull_request.number }}
           PR_BODY: "Test triggered by ${{ github.event.pull_request.html_url }}."
         run: |
-          ve1/bin/pytest tests/ --log-cli-level=WARNING --ignore=tests/functional/step_defs/test_submitted_charts.py --tb=short
+          ve1/bin/pytest tests/ -n 2 --log-cli-level=WARNING --ignore=tests/functional/step_defs/test_submitted_charts.py --tb=short
```

Comment:
Thanks for making this change, but I think this won't work because the collision on release tags and names. Currently we have `expected_tag = f'{vendor}-{chart_name}-{chart_version}'` and `expected_chart_asset = f'{vendor}-{chart_tgz}'`, and vendor is differentiated by PR number. Right now the tests won't collide between PR tests but _will_ collide within test cases of the same PR.

In order to mitigate this collision we need a unique release name and release tag even between test cases of the same PR. I think this can be a separate story after the community path is merged. WDYT @tisutisu @dperaza4dustbit  @mmulholla ?

Reply:
I think we can make the changes related to parallel running of the tests in separate PR.

Reply:
Looks like David addressed this: https://github.com/openshift-helm-charts/development/pull/66